webrtc-stats.html

```diff
@@ -2134,6 +2147,17 @@ <h2>
                   Before the first frame is encoded this attribute is missing.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>frameBitDepth</code></dfn> of type <span class="idlMemberType">unsigned
+                long</span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for video. Represents the bit depth per pixel of the raw video frame.
```

Comment:
Copy this to RTCVideoSourceStats but rephrase this sentence:
"Represents the bit depth per pixel of the last frame originating from this source" (i.e. before encoding)

And keep it here as well but rephrase:
"Represents the bit depth per pixel of the last encoded frame" (i.e. after encoding)

Reply:
Moved from outbound to RTCVideoSourceStats.